.github/copilot-instructions.md

```
@@ -1,428 +1,187 @@
+- **Framework**: SvelteKit 2 with Svelte 5 (runes mode: `$state`, `$derived`, `$effect`, `$props`)
+- **Language**: TypeScript
+- **Styling**: Tailwind CSS 4 + custom CSS design tokens
+- **Auth/DB**: Supabase (`@supabase/ssr` + `@supabase/supabase-js`)
+- **PWA**: `@vite-pwa/sveltekit`
+- **Build**: Vite 8, adapter-auto
+- **Linting**: Biome (tabs, single quotes, no trailing commas)
+- **Testing**: Vitest 4 + `@testing-library/svelte` + Playwright for E2E
 
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)